docker-compose-apm-ruby-benchmark-local-collector.yml

```diff
@@ -0,0 +1,163 @@
+x-service-resource: &service-resource
+  limits:
+    cpus: '1'
+    memory: 512M
+  reservations:
+    cpus: '1'
+    memory: 512M
+
+x-load-resource: &load-resource
+  limits:
+    cpus: '2'
+    memory: 2G
+  reservations:
+    cpus: '2'
+    memory: 2G
+
+x-collector-resource: &collector-resource
+  limits:
+    cpus: '1'
+    memory: 1G
+  reservations:
+    cpus: '1'
+    memory: 1G
+
+# this env variable is only for when EXPORT_CONTROLLER_DURATION is true (default is false)
+x-common-env-variable-controller-time: &common-env-variable-controller-time
```

Comment:
seems left over from previous attempt to have the test app's controller report duration, can this be removed?